socorro/unittest/external/es/test_index_creator.py

```diff
@@ -22,19 +20,6 @@ def __init__(self, *args, **kwargs):
 
         self.config = self.get_tuned_config(IndexCreator)
 
-    def tearDown(self):
```

Comment:
Actually, wait, ``socorro_test_temp`` is not created in this file, so why should it be removed here?